### ISSUE TITLE
Wallaby.js timeout with Visual Studio Code 1.14.x (works fine with Visual Studio Code 1.13.x)

### ISSUE THREAD

**Issue by mdebruijne** [wallabyjs/public#1234]
### Issue description or question
After the update to Visual Studio Code 1.14.x Wallaby.js doesn't work anymore. ~80% of the tests timeout with the following Wallaby.js errors;

```​​
[Error] Some long running code has been detected: one of your tests is taking more than 5000ms to execute.​​
Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
```

If I run Jest manually all tests complete without problems well within the timeout.

If I downgrade Visual Studio Code to version 1.13.x then Wallaby.js works as expected. If I upgrade Visual Studio Code to version 1.14.x then Wallaby.js breaks. 100% reproducible.

I have tried to reinstall Wallaby.js, but that doesn't resolve the problem. I am using Wallaby.js Core v1.0.455.

If you need more information then please let me know.

### Wallaby.js configuration file

```json
{
  "env": {
    "type": "node",
    "runner": "node"
  },
  "files": [
    "config/*.ts",
    "helpers/*.ts",
    "lib/*.ts",
    "routes/*.ts"
  ],
  "hints": {
    "ignoreCoverage": "istanbul ignore"
  },
  "testFramework": "jest",
  "tests": [
    "test/*.ts"
  ]
}

```

### Code editor or IDE name and version
Visual Studio Code v1.14.x

### OS name and version
Linux

**Comment by mdebruijne** [comment]
@danielepolencic I have noticed in issue #1228 that you are also having a test that times out with Visual Studio Code v1.14.x and Wallaby.js. Maybe this is the same problem. If you re-enable the test and run the test manually what is the test duration?

**Comment by ArtemGovorov** [comment]
I have tried to reproduce the issue, but everything seems t be working for me. We also have a lot of users using Jest with wallaby in VS Code 1.14.1 (and even 1.15 insiders build) without any issues. Another interesting thing is that the same wallaby.js core is working for you in one version of VS Code and not in another. This makes me think that the issue may be your project/environment specific.

Could you please share:
- a sample project, with one or two failing tests demonstrating the issue.
- open VS Code 1.14 Dev Tools (Help - Toggle Developer Tools), go to the Console, evaluate `process.env` and share the output (you may need to clear some sensitive values from it),
- also please do the same in VS Code 1.13 and compare the output for `process.env`.

**Comment by mdebruijne** [comment]
Thank you for the pointer to process.env in Visual Studio Code Developer Tools! It was exactly what I needed to analyze this further and I have found the root cause.

I am using Gentoo Linux (overlay) and with Visual Studio Code 1.14.x the Gentoo package was changed. With version 1.13.x a custom wrapper was being used to start VS Code in a shell. With version 1.14.x the package was changed to start VS Code with the Microsoft provided wrapper. The Microsoft wrapper doesn't source ~/.bashrc and therefore Google Cloud SDK variables were not available in VS Code.

Again thanks for your excellent support!
